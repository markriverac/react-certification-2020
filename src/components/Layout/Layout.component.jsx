import React from 'react';

import './Layout.styles.css';

function Layout({ children }) {
  return (
    <div className="page-container">
      <main className="container">{children}</main>
    </div>
  );
}

export default Layout;
