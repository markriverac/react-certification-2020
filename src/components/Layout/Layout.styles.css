.page-container {
  width: 100vw;
  height: 95vh;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

.container {
  width: 100vw;
  height: auto;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: center;
  overflow: auto;
}
