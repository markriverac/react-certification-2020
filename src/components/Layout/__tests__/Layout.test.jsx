import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Layout from '../Layout.component';

Enzyme.configure({ adapter: new Adapter() });

it('Renders button without exploding', () => {
  const button = renderer.create(<Layout>children</Layout>);
  const tree = button.toJSON();
  expect(tree).toMatchSnapshot();
});
