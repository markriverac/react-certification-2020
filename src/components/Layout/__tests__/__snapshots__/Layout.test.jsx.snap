// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders button without exploding 1`] = `
<div
  className="page-container"
>
  <main
    className="container"
  >
    children
  </main>
</div>
`;
